config/samples/dataplane_v1beta1_openstackdataplane_customnetworks.yaml

```diff
@@ -0,0 +1,174 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlane
+metadata:
+  name: openstack-edpm
+spec:
+  deployStrategy:
+    deploy: false
+  nodes:
+    edpm-compute-0:
+      role: edpm-compute
+      hostName: edpm-compute-0
+      ansibleHost: 192.168.1.5
+      node:
+        ansibleVars: |
+          ctlplane_ip: 192.168.1.5
+          internal_api_ip: 172.17.0.101
+          storage_ip: 172.18.0.101
+          tenant_ip: 172.10.0.101
+          external_ip: 172.20.12.76
+          fqdn_internal_api: edpm-compute-1.example.com
+          ansible_ssh_transfer_method: scp
+        ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+        ansibleUser: root
+      deployStrategy:
+        deploy: true
+  roles:
+    edpm-compute:
+      env:
+        - name: ANSIBLE_FORCE_COLOR
+          value: "True"
+        - name: ANSIBLE_ENABLE_TASK_DEBUGGER
+          value: "True"
+        - name: ANSIBLE_VERBOSITY
+          value: 4
+        - name: ANSIBLE_HOST_KEY_CHECKING
+          value: false
+      nodeTemplate:
+        managed: false
+        managementNetwork: ctlplane
+        ansibleUser: root
+        ansiblePort: 22
+        ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+        networkConfig:
+          template: |
+            ---
+            network_config:
+            - type: interface
+              name: nic2
+              mtu: 1500
+              addresses:
+                - ip_netmask:
+                    192.168.1.5/24
```

Comment:
Can we use vars in this template (eg. `{{ ctlplane_ip }}` , `{{ ctlplane_mtu}}` )

Reply:
Yeah, it can be a jinja2 template. It's just converted to a string, written to the ConfigMap and then mounted in the AEE environment. So you can provide a Jinja2 formatted template like we normally would.